### PR TITLE
Secpatch/high2

### PR DESCRIPTION
**Pull request by happythenewsad**
## WHAT
security patches, part 2 

## NOTES
rack cannot be updated with bumping rails, and our current version is the last available version before 5.x. Marking this specific issue as "won't fix" until the Rails 5 rollout. CC @brendanshean 

### Screenshots
(If applicable. Also, please censor any sensitive data)

### Notion Card Links
https://www.notion.so/quill/dependabot-patches-high-priority-part-2-2-2fc7f67313c54ab3b63e6ab1eeafbdf6

PR Checklist | Your Answer
------------ | -------------
Have you added and/or updated tests? |  not necessary
Have you deployed to Staging? | yes
Self-Review: Have you done an initial self-review of the code below on Github? | yes
Design Review: If applicable, have you compared the coded design to the mockups? | (N/A

### CODE REVIEW

**Comment by happythenewsad** [comment]
In draft until it's ancestor PR is merged.

**Comment by happythenewsad** [comment]
@cissyxyu has volunteered to merge and deploy this PR after approval, during my PTO day tomorrow. Thanks, Cissy!

File: services/QuillLMS/Gemfile.lock [hunk 1]
```diff
@@ -235,19 +236,19 @@ GEM
       google-cloud-automl-v1 (~> 0.0)
       google-cloud-automl-v1beta1 (~> 0.0)
       google-cloud-core (~> 1.5)
-    google-cloud-automl-v1 (0.3.0)
+    google-cloud-automl-v1 (0.4.0)
```
Comment (dandrabik) [inline services/QuillLMS/Gemfile.lock]: Are these google cloud updates needed?

Comment (happythenewsad) [reply services/QuillLMS/Gemfile.lock]: I'll look into this when I rebase against develop + #7770

**Review by cissyxyu — APPROVED** [review]

File: services/QuillLMS/Gemfile.lock [hunk 2]
```diff
@@ -238,16 +240,16 @@ GEM
     google-cloud-automl-v1 (0.3.0)
       gapic-common (~> 0.3)
       google-cloud-errors (~> 1.0)
-    google-cloud-automl-v1beta1 (0.4.0)
+    google-cloud-automl-v1beta1 (0.5.0)
```
Comment (happythenewsad) [inline services/QuillLMS/Gemfile.lock:243]: This gem, dependent on google-cloud-automl-v1, was updated by a  minor version ever after pinning. Should be ok.

**Review by dandrabik — APPROVED** [review]

**Pull request merged**